view/SSViewer.php

```diff
@@ -1104,7 +1104,7 @@ public function process($item, $arguments = null, $inheritedScope = null) {
 		$output = $this->includeGeneratedTemplate($cacheFile, $item, $arguments, $underlay, $inheritedScope);
 		
 		if($this->includeRequirements) {
-			$output = Requirements::includeInHTML($template, $output);
+			$output = Requirements::include_in_html($template, $output);
```

Comment:
Alternatively, we could think of `Requirements` as a special kind of template post-processor?